app/src/main/java/woowacourse/movie/MainActivity.kt

```diff
@@ -28,6 +28,13 @@ class MainActivity : AppCompatActivity() {
             insets
         }
 
+        setupListener()
+        if (savedInstanceState == null) {
+            replaceFragment<ReservationDetailsFragment>()
```

Comment:
초기에 보여줘야하는게 ReservationDeteailsFragment인데 만약 첫번째 탭이 다른 Fragment로 바뀌었을때 여기도 같이 바꾸지 못해 버그가 발생할 수 있을 것 같아요. 이렇게 변경해보면 어떨까요?
```suggestion
            binding.mainBottomNavigationBar.selectedItemId = R.id.bottom_navigation_reservation_details
```

Reply:
해당 내용 반영했습니다!

Reply:
해당 내용 반영했습니다!!

0c07b8754afdff4ae216d5f82562b4ffe18bcb59